docs/docsite/rst/network/user_guide/platform_vyos.rst

```diff
@@ -35,7 +36,7 @@ Connections Available
 .. |enable_mode| replace:: Enable Mode |br| (Privilege Escalation)
 
 
-For legacy playbooks, VyOS still supports ``ansible_connection: local``. We recommend modernizing to use ``ansible_connection: network_cli`` as soon as possible.
+The ``ansible_connection: local`` has been deprecated. Please use ``ansible_connection: network_cli`` or ``ansible_connection: netconf`` as soon as possible.
```

Comment:
Did you add netconf here on purpose?

Reply:
sorry cut n paste error. Fixed that and the wording as you suggested.